src/org/ods/component/ScanWithAquaStage.groovy

```diff
@@ -199,13 +215,28 @@ class ScanWithAquaStage extends Stage {
                 [ title: "Messages", value: prepareMessageToBitbucket(messages), ]
             ])
         }
+        if (actionableVulnerabilities?.size() > 0) {
+            if (data.messages) {
+                ((List) data.messages).add([
+                    title: "Blocking",
+                    value: "Yes"
+                ])
+            } else {
+                data.put("messages", [
+                    [
+                        title: "Blocking",
+                        value: "Yes"
+                    ]
+                ])
+            }
+        }
 
         bitbucket.createCodeInsightReport(data, context.repoName, context.gitCommit)
     }
 
     private createBitbucketCodeInsightReport(String messages) {
         String title = "Aqua Security"
-        String details = "There was some problems with Aqua:"
+        String details = "There were some problems with Aqua"
```

Comment:
Maintain the ":" at the end:
![image](https://github.com/user-attachments/assets/91833bf5-5df5-4303-9ff7-0a47b401da45)